db/migrate/20180425170810_make_nil_projects_public.rb

```diff
@@ -0,0 +1,13 @@
+class MakeNilProjectsPublic < ActiveRecord::Migration[5.1]
+  def change
+    nil_projects = Project.where(public_level_value: nil)
+
+    if nil_projects
+      nil_projects.each do |np|
+        np.update_attribute(:public_level_value, 'public')
+      end
+    end
```

Comment:
It's fine for this migration, but for future reference, this could have been a one-liner:

`Project.where(public_level_value: nil).update_all(public_level_value: 'public')`

Reply:
👍